### PR TITLE
Cem 2595 info header

### PR DESCRIPTION
**Pull request by Hunterjdf**
Simple component, Let me know if there is are issues with how it is constructed.

### CODE REVIEW

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Where is picture in PR

File: src/Containers/InfoHeader/InfoHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    Text: String;
+}
+
+export const InfoHeader: React.FC<InfoHeaderProps>  = ({
+    Text,
+    ...props
+}): React.ReactElement => (
+    <InfoHeaderStyle>
+    {Text} <Icon />
+</InfoHeaderStyle>
```
Comment (ralph-dev) [inline src/Containers/InfoHeader/InfoHeader.tsx:16]: Fix indentation

File: src/Containers/InfoHeader/InfoHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export interface InfoHeaderProps extends React.HTMLAttributes<HTMLDivElement> {
+    /* Text that will be shown */
+    Text: String;
+}
+
+export const InfoHeader: React.FC<InfoHeaderProps>  = ({
+    Text,
+    ...props
+}): React.ReactElement => (
+    <InfoHeaderStyle>
```
Comment (ralph-dev) [inline src/Containers/InfoHeader/InfoHeader.tsx:14]: Spread props

**Comment by Hunterjdf** [comment]
image
Component Image

**Comment by Hunterjdf** [comment]
image

**Comment by Hunterjdf** [comment]
Zoomed in view. 
- Updated with spread header props

File: src/Containers/InfoHeader/InfoHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    infotext,
+    ...props
+}): React.ReactElement => (
+    <InfoHeaderStyle {...props}>
+        {infotext} 
+        <Icon />
+    </InfoHeaderStyle>
+);
+
+const InfoHeaderStyle = styled(Heading)`
```
Comment (ralph-dev) [inline src/Containers/InfoHeader/InfoHeader.tsx:21]: It's not a Style, it's a text

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Waiting for in meeting changes

**Comment by Hunterjdf** [comment]
Sorry sir, in regards to in meeting changes are you referencing the changes we made during the meeting or are referencing something else ? 
I think I may have missed something that you said, while focusing on the header props.  
Aside from the fixed comment, what was it that I didn't get?

**Review by ralph-dev — APPROVED** [review]
Lgtm

**Pull request merged**